src/components/sdk/MeetingControls/VideoInputBackgroundBlurControl.tsx

```diff
@@ -88,10 +103,13 @@ const VideoInputBackgroundBlurControl: React.FC<Props> = ({
             const receivedDevice = videoInputSelectionToDevice(option.deviceId);
             if ('chooseNewInnerDevice' in activeVideoDevice) {
               // @ts-ignore
-              const transformedDevice = activeVideoDevice.chooseNewInnerDevice(receivedDevice);
+              const transformedDevice =
+                activeVideoDevice.chooseNewInnerDevice(receivedDevice);
```

Comment:
I think the build script failed because the `// @ts-ignore` should be moved one line below after `const transformedDevice =` and above `activeVideoDevice.chooseNewInnerDevice(receivedDevice);`.

```
const transformedDevice =
  // @ts-ignore
  activeVideoDevice.chooseNewInnerDevice(receivedDevice);
```